libraries/stdlib/genosl/mx_length_unit.osl

```diff
@@ -0,0 +1,37 @@
+
+float unit_ratio(int unit_from, int unit_to) {
+
+    #define LENGTH_UNITS 10
+    // based on unittypedef name="length" in stdlib_defs.mtlx
+    // nanometer, millimeter, micron, centimeter, foot, yard, kilometer, inch, mile, meter
+    float u_length_unit_scales[LENGTH_UNITS] = {
+        1000000000.0,
+        1000.0,
+        1000000.0,
+        100.0,
+        3.28099990,
+        1.09361303,
+        0.00100000005,
+        39.3699989,
+        0.000621000014,
+        1.0
+    };
```

Comment:
I assume this will be moved out to be a global that is generated vs hard-coded? And this will work be the same for GLSL as well?

Reply:
Yes